tests/Cache/QueryCacheProfileTest.php

```diff
@@ -135,4 +135,29 @@ public function testShouldGenerateSameKeysIfNoneOfTheParamsChanges(): void
 
         self::assertEquals($firstCacheKey, $secondCacheKey, 'Cache keys should be the same');
     }
+
+    public function testShouldGenerateDifferentPasswordInTheParams(): void
+    {
+        [, $firstRealCacheKey] = $this->queryCacheProfile->generateCacheKeys(
+            $this->query,
+            $this->params,
+            $this->types,
+            [
+                'user'     => 'database_user',
+                'password' => 'first-password',
+            ]
+        );
+
+        [, $secondRealCacheKey] = $this->queryCacheProfile->generateCacheKeys(
+            $this->query,
+            $this->params,
+            $this->types,
+            [
+                'user'     => 'database_user',
+                'password' => 'second-password',
+            ]
+        );
+
+        self::assertEquals($firstRealCacheKey, $secondRealCacheKey, 'Cache keys for different password should be the same');
```

Comment:
This should make the coding standard tool happy:
```suggestion
        self::assertEquals(
            $firstRealCacheKey,
            $secondRealCacheKey,
            'Cache keys for different password should be the same'
        );
```